sdk/reservations/Azure.ResourceManager.Reservations/src/Generated/Models/ReservationCatalog.cs

```diff
@@ -51,31 +51,25 @@ internal ReservationCatalog(string appliedResourceType, string skuName, IReadOnl
             Capabilities = capabilities;
         }
 
-        /// <summary> The type of resource the SKU applies to. </summary>
+        /// <summary> The type of resource the sku applies to. </summary>
         public string AppliedResourceType { get; }
-        /// <summary> The name of SKU. </summary>
+        /// <summary> The name of sku. </summary>
         public string SkuName { get; }
-        /// <summary> The billing plan options available for this SKU. </summary>
+        /// <summary> The billing plan options available for this sku. </summary>
         public IReadOnlyDictionary<string, IList<ReservationBillingPlan>> BillingPlans { get; }
         /// <summary> Available reservation terms for this resource. </summary>
         public IReadOnlyList<ReservationTerm> Terms { get; }
         /// <summary> Gets the locations. </summary>
         public IReadOnlyList<AzureLocation> Locations { get; }
         /// <summary> Gets the sku properties. </summary>
         public IReadOnlyList<SkuProperty> SkuProperties { get; }
-        /// <summary> Pricing information about the SKU. </summary>
-        internal CatalogMsrp Msrp { get; }
-        /// <summary> Amount in pricing currency. Tax not included. </summary>
-        public PurchasePrice MsrpP1Y
-        {
-            get => Msrp?.P1Y;
-        }
```

Comment:
Please create a partial class of this in the Custom/Models directory and add this property back